packages/ui/src/Button.tsx

```diff
@@ -110,11 +130,14 @@ export const Button = forwardRef<HTMLElement, ButtonProps>(
       iconRightSize,
       iconRightColor,
       iconRightClassName,
+      rel = 'noopener',
+      target,
       ...rest
     },
     ref,
   ) => {
     const tooltipId = useUID()
+    const relFinal = useDeepCompareMemo(() => (Array.isArray(rel) ? rel.join(' ') : rel), [rel])
```

Comment:
@jvorcak I am not sure if `useMemo` makes sense here. The result is a string. There is no such thing as a reference to a string in JavaScript, right? So we can easily re-compute this string on every re-render and pass it to `Component`. If string value is the same it won;t cause the `Component` to re-render.

Reply:
@aigoncharov I think you're right, I took this rom `Link` component, but to be honest, I haven't thought of this more in depth. I assume I should remove this from Link also?

Reply:
@jvorcak probably, I did not think it through back then while implementing it in the `Link` :) My bad.

Reply:
But I'm thinking ..., since the parameter can also be an array, memoization actually might make sense here. I'll follow up on Slack.